index.js

```diff
@@ -1,9 +1,12 @@
+const REPLIT_URL = "FlagClicked.9gr.repl.co"; // Please change this URL to your replit URL before starting!
```

Comment:
I kinda wanted to make the url figure itself out... Maybe express has something that could tell us what domain it's running on?

Reply:
doesnt seem there is.. there is a hacky method for this